job-engine/app/web/src/main/java/org/eclipse/kapua/job/engine/app/web/JobEngineApplication.java

```diff
@@ -49,31 +45,7 @@ protected void configure() {
         property(ServerProperties.WADL_FEATURE_DISABLE, true);
         register(UriConnegFilter.class);
         register(JacksonFeature.class);
-
-        register(new ContainerLifecycleListener() {
-
```

Comment:
The lifecycle listener is no longer needed: the jaxbcontext is now initialized in the main DI context (guice) and injected in jetty's own DI in JaxbContextResolver's new implementation